websites/D/dod Games/presence.ts

```diff
@@ -0,0 +1,254 @@
+				stateString += `${value.winner ? " 🏆 " : ""}${key}: ${value.score} – `;
+			}
+			if (Object.keys(state).length === nPlayers || gameOverVisible)
+				presenceData.state = stateString.substring(0, stateString.length - 3);
+			else {
+				presenceData.state = `${
+					Object.keys(state).length
+				}/${nPlayers} παίκτες...`;
+				presenceData.buttons = [
+					{
+						label: "Join Game",
+						url: url.textContent,
+					},
+				];
+			}
+			presenceData.details = `Παίζει Αγωνία | 💪🏻 ${document
+				.querySelector(".mytr")
+				.getAttribute("elo")}`;
+			presenceData.startTimestamp = timeStarted;
+			break;
+		} else if (url.textContent.includes("game=tichu&room=")) {
+			// Tichu
+			if (!isInGame) {
+				timeStarted = Date.now();
+				isInGame = true;
+			}
+
+			const teamPlayers: string[] = [],
+				opPlayers: string[] = [],
+				playerState: { [key: string]: TichuPlayerState } = {},
+				bot = document.querySelector(
+					"#nickholder_bottom .playerName"
+				).textContent,
+				up = document.querySelector("#nickholder_up .playerName").textContent,
+				left = document.querySelector(
+					"#nickholder_left .playerName"
+				).textContent,
+				right = document.querySelector(
+					"#nickholder_right .playerName"
+				).textContent;
+
+			if (bot) playerState.bot = { name: bot, bet: null };
+			if (up) playerState.up = { name: up, bet: null };
+			if (left) playerState.left = { name: left, bet: null };
+			if (right) playerState.right = { name: right, bet: null };
+
+			for (const pos of ["up", "left", "right"]) {
+				if (!(pos in playerState)) continue;
+				const betElement = document.querySelector<HTMLDivElement>(
+					`#nickholder_${pos} #tichugrand`
+				);
+				if (betElement.style.display !== "none")
+					playerState[pos].bet = betElement.className;
+			}
+
+			let betElement =
+				document.querySelector<HTMLDivElement>("#btnTichuToggle");
+			if (betElement.style.display !== "none") playerState.bot.bet = "tichu";
+
+			betElement = document.querySelector<HTMLDivElement>("#btnGrandToggle");
+			if (betElement.style.display !== "none") playerState.bot.bet = "grand";
+
+			for (const pos in playerState) {
+				const player = playerState[pos];
+				(["bot", "up"].includes(pos) ? teamPlayers : opPlayers).push(
+					`${
+						player.bet === "tichu" ? "🟠" : player.bet === "grand" ? "🔴" : ""
+					}${player.name}`
+				);
+			}
+
+			if (teamPlayers.length === 2 && opPlayers.length === 2) {
+				presenceData.state = `(${teamPlayers[0]}, ${teamPlayers[1]}) ${
+					document.querySelector("#txtMyTeamScore").textContent
+				} – ${document.querySelector("#txtOpTeamScore").textContent} (${
+					opPlayers[0]
+				}, ${opPlayers[1]})`;
+			} else {
+				presenceData.state = `${
+					teamPlayers.length + opPlayers.length
+				}/4 παίκτες...`;
+				presenceData.buttons = [
+					{
+						label: "Join Game",
+						url: url.textContent,
+					},
+				];
+			}
+			presenceData.details = `Παίζει Tichu | 💪🏻 ${document
```

Comment:
if it is all greek on their site, please confirm "Tichu" is the same.

Reply:
You are right! That's in English, so I guess it's ok to have it in English, too?
![image](https://user-images.githubusercontent.com/16375254/177019504-063c78de-c2be-429d-8dbb-59dcc0357b99.png)

Reply:
Perfect. If they're in English, they say English. If they're Greek, they stay Greek.